stream_alert/apps/_apps/README.rst

```diff
@@ -96,3 +96,66 @@ back on local system
 
   # scp to local host's current directory
   $ scp -i /path/to/<private-key>.pem ec2-user@public.dns.address:~/pip_temp/pip.zip .
+
+Build dependencies on an virtual machine locally (python3.7)
+============================================================
+
+There is a Vagrantfile located in streamalert repo root directory. We can use this Vagrantfile to spin up an Ubuntu 18.04 virtual machine locally to build dependencies for box sdk or aliyun sdk.
```

Comment:
please stylize `streamalert` in docs properly as `StreamAlert`. you can also use a link here to the Vagrantfile like:
```rst
`Vagrantfile <https://github.com/airbnb/streamalert/blob/master/Vagrantfile>`_
```

Reply:
suggested complete verbiage:

```rst
There is a `Vagrantfile <https://github.com/airbnb/streamalert/blob/master/Vagrantfile>`_ located in the root of the StreamAlert repository. This file can be used to spin up a virtual machine and build dependencies for the box sdk or aliyun sdk.
```

Reply:
Sweet!